Project-Aurora/Project-Aurora/Devices/Uniwill/KeyboardDevice.cs

```diff
@@ -0,0 +1,346 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Drawing;
+using System.Linq;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using Aurora.Devices.RGBNet;
+using Aurora.Settings;
+using Aurora.Utils;
+using IronPython.Runtime.Operations;
+using keyboard;
+using Microsoft.Win32;
+using RegistryUtils;
+
+namespace Aurora.Devices.Uni
+{
+    
+    public class KeyboardDevice : Device
```

Comment:
Please auto format this file. The class should be renamed to be more descriptive. KeyboardDevice gives me no useful information.

Reply:
Got it, thanks. I will change to our name.